src/test/scala/org/broadinstitute/dsde/firecloud/webservice/NihApiServiceSpec.scala

```diff
@@ -288,21 +288,12 @@ class NihApiServiceSpec extends ApiServiceSpec with BeforeAndAfterAll with SamMo
     services =>
       Post("/sync_whitelist") ~> sealRoute(services.syncRoute) ~> check {
         status should equal(NoContent)
-        assertSameElements(Set(services.thurloeDao.TCGA_AND_TARGET_LINKED, services.thurloeDao.TCGA_LINKED),
-                           services.samDao.groups(tcgaDbGaPAuthorized).map(_.value)
-        )
-        assertSameElements(Set(services.thurloeDao.TCGA_AND_TARGET_LINKED, services.thurloeDao.TARGET_LINKED),
-                           services.samDao.groups(targetDbGaPAuthorized).map(_.value)
-        )
```

Comment:
I removed these checks and the on below because they only test that we have our mocks setup correctly which is not useful. The underlying function is well tested in unit tests.